apps/smartling/lambda/src/index.ts

```diff
@@ -4,6 +4,15 @@ import * as express from 'serverless-express/express';
 import * as path from 'path';
 import fetch from 'node-fetch';
 import { Issuer } from 'openid-client';
+import { URL } from 'url';
+
+function requestProxier(proxyUrl: string, proxyPathPrefix = '') {
```

Comment:
Does that work so easy? Our frontend is using webpack which (afaik) sets up a web socket to the backend. I doubt that `fetch` could handle that kind of request.

Or is this something we kind of ignore for now as it also doesn't work with the existing setup?

Reply:
It works just fine actually (again, for development). But let me know if you're driving at something else, since I might misunderstand your question.